packages/react-components/src/components/bar-chart/barChart.tsx

```diff
@@ -56,9 +59,33 @@ export const BarChart = (props: BarChartProps) => {
     ...rest
   } = props;
 
+  const {
+    viewport: injectedViewport,
+    setViewport,
+    group,
+    lastUpdatedBy,
+  } = useViewport();
+  const viewport = passedInViewport || injectedViewport || DEFAULT_VIEWPORT;
+
+  const alarmQueries = getAlarmQueries(queries);
+  const timeSeriesQueries = getTimeSeriesQueries(queries);
+
+  const mapAlarmQueriesToRequests = alarmQueries.flatMap((query) =>
+    convertAlarmQueryToAlarmRequest(query)
+  );
+
+  const transformedAlarms = useAlarms({
+    iotSiteWiseClient: alarmQueries[0]?.iotSiteWiseClient,
```

Comment:
A safer way to access the query would be:
```
alarmQueries.find(Boolean)
```

This gets you the first item on the array if it exists

Reply:
Or
```
alarmQueries.at(0)
```